cirq-superstaq/cirq_superstaq/sampler.py

```diff
@@ -50,15 +50,12 @@ def __init__(
         service: css.service.Service,
         target: str,
     ) -> None:
-        """Constructs the sampler. Uers should get a sampler from the `sampler` method on
+        """Constructs the sampler. Users should get a sampler from the `sampler` method on
         `css.Service`.
```

Comment:
Should be one line, e.g., 
```suggestion
        """Constructs the sampler, accessed from the `sampler` method on `css.Service`.
```

Reply:
Done.